v0/destinations/awin/utils.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Returns final params
+ * @param {*} params
+ * @param {*} advertiserId
+ * @returns params
+ */
+const getParams = (params, advertiserId) => {
+  params.tt = "ss";
+  params.tv = "2";
+  params.merchant = advertiserId;
+  params.ch = "aw";
+  if (params.amount) {
+    if (!params.commissionGroup) {
+      params.commissionGroup = "DEFAULT";
+    }
+    params.parts = String(params.commissionGroup).concat(
+      ":",
+      String(params.amount)
+    );
+    delete params.commissionGroup;
+  }
+  if (!params.cks) {
+    params.cks = "awc";
```

Comment:
Why are we setting it to some default value?

Reply:
Removed the default value